content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,123 @@
+# data status
+
+Show changes in data tracked by DVC between the last Git commit, the metafiles
+and the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+Shows changes that have differences between the last Git commit (`HEAD`) and
+committed data specified in [metafiles](/doc/user-guide/project-structure),
+uncommitted changes that have differences between that specified in the
+metafiles and the <abbr>workspace</abbr>, and the changes that are not in the
+cache but is specified in those metafiles.
+
+The first change shows what is already committed to DVC via `dvc add` or
```

Comment:
As I mentioned below in the `#Output` section ppl won't be able to understand this paragraph. It's too technical, it's trying to explain in some abstract terms and very technical terms quite complex things. I would get rid / combine the the output description (see also the comment below)